<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToFriendMultiSessionDialog</class>
 <widget class="QDialog" name="ToFriendMultiSessionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>387</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QPushButton#CallButton
{
color: white;
font: bold 14px;
border-width: 10;
border-image: url(:/MyP2PWeb/Resources/button_green_normal.png) 10;
padding: -10;
}
QPushButton#CallButton:pressed
{
border-width: 10;
border-image: url(:/MyP2PWeb/Resources/button_green_pressed.png) 10;
padding: -10;
}
QPushButton#HangUpButton
{
color: white;
font: bold 14px;
border-width: 10;
border-image: url(:/MyP2PWeb/Resources/button_green_normal.png) 10;
padding: -10;
}
QPushButton#HangUpButton:pressed
{
border-width: 10;
border-image: url(:/MyP2PWeb/Resources/button_green_pressed.png) 10;
padding: -10;
}
QLabel#PermissionLabel
{
font: bold 14px;
}
QLabel#MuteMicrophoneLabel
{
font: bold 12px;
}
QLabel#MuteSpeakerLabel
{
font: bold 12px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="TitleBarWidget" name="m_TitleBarWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>130</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="StyledDlgItemsBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QFrame" name="m_OffersFrame">
        <property name="styleSheet">
         <string notr="true">QFrame#m_OffersFrame
{   
background-color: #e0f9f9;
border: 1px solid blue;
border-radius: 4px; 
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="VxPushButton" name="m_VoipButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="VxPushButton" name="m_VideoChatButton">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="VxPushButton" name="m_TrueOrDareButton">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="VxPushButton" name="m_SendAttachmentButton">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="VxPushButton" name="m_OptionsButton">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="m_ResponseFrame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="VxPushButton" name="m_AcceptOfferButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="m_AcceptLabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="VxPushButton" name="m_RejectOfferButton">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="m_RejectLabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="VidWidget" name="m_VidWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>240</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="TodGameWidget" name="m_TodGameWidget" native="true"/>
      </item>
      <item>
       <widget class="QFrame" name="m_HangupSessionFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame#m_HangupSessionFrame
{   
background-color: #e0f9f9;
border: 1px solid blue;
border-radius: 4px; 
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="VxPushButton" name="m_HangupSessionButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="SessionWidget" name="m_SessionWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="BottomBarWidget" name="m_BottomBarWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>0</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitleBarWidget</class>
   <extends>QWidget</extends>
   <header>TitleBarWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BottomBarWidget</class>
   <extends>QWidget</extends>
   <header location="global">BottomBarWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VxPushButton</class>
   <extends>QPushButton</extends>
   <header>VxPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>SessionWidget</class>
   <extends>QWidget</extends>
   <header>SessionWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VidWidget</class>
   <extends>QWidget</extends>
   <header location="global">VidWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TodGameWidget</class>
   <extends>QWidget</extends>
   <header>TodGameWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
